newjersey/navigator.business.nj.gov
# feat: [#187651449] remove ada-compliant from CMS and consolidate to ada

## Description

We wanted to remove the contextual info element ada-complaint and replace all used instances with ada

(just so happens that theres only one from what I can tell)



### Ticket



This pull request resolves [#187651449](https://www.pivotaltracker.com/story/show/187651449).

### Approach



### Steps to Test



Global search the code base for ada-compliant and see that it doesn't exist anywhere

Go to the one instance of tenant-improvements and make sure it now refers to the correct ada object (can be done by registering as a canabis business and then going to tenant-improvments in section 3 of the roadmap)

### Notes



## Code author checklist
